src/display/text_layer.js

```diff
@@ -65,6 +65,9 @@ const ascentCache = new Map();
 let _canvasContext = null;
 
 function getCtx(lang = null) {
+  if (_canvasContext && _canvasContext.canvas.lang !== (lang || "")) {
+    cleanupTextLayer();
```

Comment:
This function should probably be renamed to `cleanupTextLayerCanvas` or similar. While reading this diff I was very confused as to why when we have to render some text with a different language we need to cleanup the whole text layer.